apps/src/templates/sectionAssessments/MatchAssessmentsOverviewTable.jsx

```diff
@@ -0,0 +1,241 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import {Table, sort} from 'reactabular';
+import {tableLayoutStyles, sortableOptions} from '../tables/tableConstants';
+import i18n from '@cdo/locale';
+import wrappedSortable from '../tables/wrapped_sortable';
+import orderBy from 'lodash/orderBy';
+import MultipleChoiceAnswerCell from './MultipleChoiceAnswerCell';
+import styleConstants from '@cdo/apps/styleConstants';
+import {setQuestionIndex} from './sectionAssessmentsRedux';
+import ReactTooltip from 'react-tooltip';
+import {optionDataPropTypeMatch} from './assessmentDataShapes';
+
+export const COLUMNS = {
+  OPTION: 0
+};
+
+const ANSWER_COLUMN_WIDTH = 100;
+const PADDING = 20;
+
+const styles = {
+  answerColumnHeader: {
+    width: ANSWER_COLUMN_WIDTH,
+    textAlign: 'center'
+  },
+  answerColumnCell: {
+    width: ANSWER_COLUMN_WIDTH,
+    padding: 0,
+    height: 40
+  },
+  notAnsweredCell: {
+    padding: 0,
+    height: 40
+  },
+  questionCell: {
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap'
+  }
+};
+
+const NOT_ANSWERED = 'notAnswered';
+
+const answerColumnsFormatter = (
+  percentAnswered,
+  {rowData, columnIndex, rowIndex, property}
+) => {
+  let percentValue = 0;
+  const answerResults = rowData.answers[columnIndex - 1] || {};
+
+  if (property === NOT_ANSWERED) {
+    percentValue = Math.round(
+      (rowData.notAnswered / rowData.totalAnswered) * 100
+    );
+  } else {
+    percentValue = Math.round(
+      (answerResults.numAnswered / rowData.totalAnswered) * 100
+    );
+  }
+
+  return (
+    <MultipleChoiceAnswerCell
```

Comment:
Do we want to give this component a more generic name since it's now being used for both MC and Match?

Reply:
I don't think we currently have a good name for "multiple choice or match question", but this could be a great time to make one up!